### PR TITLE
Add .subheader class

### PR DESCRIPTION
**Pull request by Scotchester**
This came about via cfpb/design-manual#273
## Adds:
- A `.subheader` class that styles text at the same size at 18px with the regular weight of the webfont.
## Review:
- @cfpb/front-end-team-admin 
- @mebates
## Preview:

screen shot 2015-05-04 at 13 46 45

### CODE REVIEW

**Comment by Scotchester** [comment]
Can I get a review on this, please? It won't take long :)

**Comment by ascott1** [comment]
Looks good to me. My only question/comment is are we still manually entering the web font names rather than using a variable?

**edit:** If so, this is good to merge, but we should revisit that in a separate issue.

**Comment by Scotchester** [comment]
You're looking at the compiled CSS :)

https://github.com/cfpb/cf-core/pull/34/files#diff-102e49d7597956453eba4e161d921164R263

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-core",
-  "version": "0.9.0",
+  "version": "0.9.1",
```
Comment (anselmbradford) [inline bower.json]: Changelog shows this PR as `0.10.0`

File: src/cf-base.less [hunk 2]
```diff
@@ -246,6 +250,19 @@ h6,
...
 
+.subheader {
+    // For longer subheaders that we don't want to be too large or heavier than
+    // the regular weight.
+    @font-size: 18px;
+
+    margin-top: 0;
+    // Spacing in the design manual is based off of the baseline so this margin
+    // number has been eyeballed to be 30px.
+    margin-bottom: unit( 21px / @font-size, em );
```
Comment (anselmbradford) [inline src/cf-base.less:261]: With settings that are being optically aligned ("eyeballed"), what do you think of formatting the units so they are in the format of `unit( (base - offest) / font-size, unit);`? That way it's clearer what the intended output pixel dimensions are and it'll be easier to spot where different pieces of code share the same margin width, etc. Something like:

``` less
margin-bottom: unit( (30px - 9px) / @font-size, em );
```

Comment (Scotchester) [inline src/cf-base.less:261]: Happy to do it, but I'd like to do it everywhere in this file/project at once, and would prefer to tackle it in a later PR.

File: bower.json (hunk 1 above)
Comment (Scotchester) [inline bower.json]: Good catch! Fixed.

File: src/cf-base.less (hunk 2 above)
Comment (anselmbradford) [inline src/cf-base.less:261]: That's fine with me.

**Comment by anselmbradford** [comment]
Review done :+1:

**Comment by Scotchester** [comment]
Thanks, y'all!

**Pull request merged**